src/collections/_documentation/sdks/javascript/errors/breadcrumbs.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Breadcrumbs
+excerpt: ""
+---
+
+{%- capture __errors_manual_breadcrumbs -%}
+
+This example is for an application that sometimes throws an error after the screen resizes.
+
+
+```js
+
+window.addEventListener('resize', function(event){
```

Comment:
Code like this should never be used in production, as it'll emit hundreds of breadcrumbs if someone drags the window to change its size, but conceptually it's valid. We can leave a comment of sort:

> Ideally with an appropriate debouncing mechanism, to not emit tons of breadcrumbs

Reply:
Would it be better if we changed the example?

Reply:
Proposed an updated example